python/spacewalk/spacewalk-backend.changes

```diff
@@ -1,3 +1,7 @@
+- Exclude invalid mirror urls for reposync (bsc#1203826)
+
+------------------------------------------------------------------
+
```

Comment:
This is not needed, you just need to put your new entry on top of the existing ones:

```suggestion
```